docusaurus/sidebars.js

```diff
@@ -258,7 +258,7 @@ const connectorCatalog = {
       link: {
         type: "generated-index",
       },
-      items: [sourcePostgres, getSourceConnectors()],
+      items: [sourcePostgres, ...getSourceConnectors()].sort((itemA, itemB) => itemA.label.localeCompare(itemB.label)),
```

Comment:
It should not be needed anymore to specify `sourcePostgres` explicitly if we anyway sort it alphabetically. The subpages of Postgres should be loaded automatically (so no need to explicitly specify them). Only problem seems (which we should maybe then fix as part of that PR), that the sidebar vanishes for some reason when being on a postgres subpage if you remove it explicitly here (but we should anyway fix that, since it also happens on some other pages).